modules/4337/contracts/Safe4337Module.sol

```diff
@@ -212,9 +212,17 @@ contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandle
     }
 
     /**
-     * @dev Checks if the signatures length is correct and does not contain additional bytes. The function does not
+     * @notice Checks if the signatures length is correct and does not contain additional bytes. The function does not
      * check the integrity of the signature encoding, as this is expected to be checked by the {Safe} implementation
      * of {checkSignatures}.
+     * @dev A malicious bundler can pad additional bytes to the `signatures` data, causing the account to pay more gas
+     * than needed for user operation validation. Safe account has two types of signatures: EOA and Smart Contract
```

Comment:
I think this sentence needs to be clarified. To me it reads like it is currently possible (which AFAIU, it isn't in the case for smart contract verifiers that have signature length checks).

Reply:
updated https://github.com/safe-global/safe-modules/pull/471/commits/36c0742db90438c850b1cf19403e0fb346418fec